config/index.js

```diff
@@ -182,7 +182,7 @@ var conf = convict({
     forcedEmailAddresses: {
       doc: 'force usage of fxa-email-service when sending emails to addresses that match this pattern',
       format: RegExp,
-      default: /emailservice.[A-Za-z0-9._%+-]+@restmail.net$/,
+      default: /emailservice\.[A-Za-z0-9._%+-]+@restmail\.net$/,
```

Comment:
Seems like whether this PR is approved or not, this change should make it's way into master.